scl/darwinosl/plugin.conf

```diff
@@ -0,0 +1,79 @@
+#############################################################################
+# Copyright (c) 2023 Hofi <hofione@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+# NOTE: For predicate hints see:
+#						log help predicates
+@define def-osl-predicate "    (eventType == 'logEvent' || eventType == 'lossEvent' || eventType == 'stateEvent' || eventType == 'userActionEvent')
+													  && (logType != 'debug')
+													"
+
+block source darwin-oslog(...) {
+@requires darwinosl "The darwinosl() source driver depends on the darwinosl module"
+	darwinosl(
+		filter-predicate("`def-osl-predicate`")
+		`__VARARGS__`
+		flags(syslog-protocol dont-store-legacy-msghdr assume-utf8)
+	);
+};
```

Comment:
I think you could use set defaults from the C code, I would not complicate it by using a block as well.

Reply:
I wanted to expose these here to
- give a properly visible overview, hint about the default to the users
- prevent overwriting of flags(syslog-protocol dont-store-legacy-msghdr assume-utf8) defaults